src/ResourceManager/DataFactories/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Fix credential encryption issue that caused no meaningful error for some encryption operations.
```

Comment:
@skywalker-wu please update the DataFactoryV2 [change log](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/DataFactories/Commands.DataFactoryV2/Changelog.md) as well

Reply:
Sure, updated.